README.md

```diff
@@ -232,23 +234,9 @@ building host and profile configurations
 ...Building host configuration for "controller-0"
 ...Building host profile configuration for "controller-0"
 ...Running profile filters for "controller-0-profile"
-...Building host configuration for "controller-1"
```

Comment:
Please keep these outputs regarding the other hosts. You're not getting these because you didn't have those hosts

Reply:
Updated